src/website/challonge.ts

```diff
@@ -373,23 +373,16 @@ export class WebsiteWrapperChallonge implements WebsiteWrapper {
 		return {
 			player1: match.player1_id,
 			player2: match.player2_id,
-			matchId: match.id
+			matchId: match.id,
+			round: match.round
 		};
 	}
 
-	public async getMatches(tournamentId: string): Promise<WebsiteMatch[]> {
-		const webMatches = await this.indexMatches(tournamentId, "open");
-		// don't need to check length like below function because a 0-length array is valid
+	public async getMatches(tournamentId: string, open = false, playerId?: number): Promise<WebsiteMatch[]> {
+		const webMatches = await this.indexMatches(tournamentId, open ? "open" : undefined, playerId);
 		return webMatches.map(this.wrapMatch.bind(this));
```

Comment:
This method doesn't actually refer to `this` so you don't need to `.bind` at all!

Reply:
Good catch, force of habit/overcorrection from early design woes